example/http/request.cpp

```diff
@@ -30,6 +32,43 @@ static bool set_header(http::Headers &headers, const std::string option) {
     return true;
 }
 
+static std::function<http::Response(net::Buffer)> http_read_response_coro() {
```

Comment:
Note: of course this could made be more robust to stop after a single message has been read.